### ISSUE TITLE
when -T or -R is specified, really return the specific times mentioned

### ISSUE THREAD

**Issue by stefco** [stefco/geco_data#14]
delete the first and last lines as needed to return the correct time interval.

**Comment by stefco** [comment]
- [ ] will need to check the bitrate to figure out the proper number of bits to remove

use the following code to cut `${terms_cut_from_start}` until the end of a particular line of values:

``` awk
awk -F',' '{for(i='${terms_cut_from_start}'+1;i<NF;i++){printf "%s,", $i}; i=NF; printf "%s", $i; printf "\n"}'
```

use the following to cut `${terms_cut_from_end}` from the end of a particular line of values:

``` awk
awk -F',' '{for(i=1;i<NF-'${terms_cut_from_end}';i++){printf "%s,", $i}; i=NF-'${terms_cut_from_end}'; printf "%s", $i; printf "\n"}'
```

these were useful:
http://stackoverflow.com/questions/2961635/using-awk-to-print-all-columns-from-the-nth-to-the-last
http://stackoverflow.com/questions/26842504/awk-command-line-printing-column-separated-by-comma

**Comment by stefco** [comment]
more nicely formatted:

``` awk
cut_terms_from_start () {
    awk -F',' '{
        for(i='$1'+1;i<NF;i++){
            printf "%s,", $i
        };
        i=NF;
        printf "%s", $i;
        printf "\n"
    }'
}
```

``` awk
cut_terms_from_end () {
    awk -F',' '{
        for(i=1;i<NF-'$1';i++){
            printf "%s,", $i
        };
        i=NF-'$1';
        printf "%s", $i;
        printf "\n"
    }'
}
```

**Comment by stefco** [comment]
Ended up doing this more efficiently by putting output on newlines, making it way easier to `sed` it up. Wrote a couple of functions to handle this. Should be extensible to arbitrarily long time periods, since it applies the stripping to each 64 second segment individually.

**Comment by stefco** [comment]
Implemented in 75ea7b2.

**Comment by stefco** [comment]
Not quite working yet for dump from frame.

You can override `noclobber` by replacing `>` with `>|`:
http://www.cyberciti.biz/tips/howto-keep-file-safe-from-overwriting.html

**Comment by stefco** [comment]
Issues fixed in 8a2cc67
